### DISCUSSION TITLE
Code search always failed today with `Search failed. Please try again later.`

### DISCUSSION THREAD

**Discussion by yyjdelete** [community/community#127206] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Code search always failed today with `Search failed. Please try again later.`
But https://www.githubstatus.com/ say everything works well.

![image](https://github.com/community/community/assets/1819074/ea04530f-739a-4836-9326-61781961ac4f)

**Comment by LiteBrite82** [comment]
Hi @yyjdelete 

I was not able to replicate the issue: using [`org:community test`](https://github.com/search?q=org%3Acommunity%20test&type=code)

Is this still an issue?

**Comment by yyjdelete** [comment]
@LiteBrite82 Thanks, it works well now.

**Comment by iCodeOfTruth** [comment]
Hey, this just happened to me, how did you fix it?

**Comment by smasongarrison** [comment]
It also just happened to me

**Reply by amakhrov** [reply]
happenning across all repos, both public and private

**Reply by xtqqczze** [reply]
There is an [ongoing incident that is affecting code search](https://www.githubstatus.com/incidents/myvz2tsj2dh8).

**Comment by Choons** [comment]
Search is broken today?

**Reply by cschill2020** [reply]
Same for me

**Comment by giuliano-macedo** [comment]
Sextou :beers:

**Comment by omartabibi** [comment]
facing the same problem too

**Reply by defonotxspy** [reply]
everyone is facing this problem, i think it will be fixed later today

**Comment by olsenbudanur** [comment]
same issue, pls fix. Monorepo too big to index/search locally :( 🙏

**Reply by giuliano-macedo** [reply]
Just a tip: maybe try out `ag`:
https://www.cyberciti.biz/open-source/command-line-hacks/ag-supercharge-string-search-through-directory-hierarchy/#attachment_1099

**Comment by LiteBrite82** [comment, marked as answer]
@BigPanda97 @olsenbudanur @defonotxspy @remoteagle @giuliano-macedo @cschill2020 @Choons @amakhrov @smasongarrison @xtqqczze 
Yep. I have a pinned post dedicated to it [GitHub Code Search is experiencing a disruption [ Search failed. Please try again later ]- 10/11/2024 #141212](https://github.com/orgs/community/discussions/141212)

**Comment by defonotxspy** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> Code search always failed today with `Search failed. Please try again later.` But https://www.githubstatus.com/ say everything works well.
> 
> ![image](https://private-user-images.githubusercontent.com/1819074/336430688-ea04530f-739a-4836-9326-61781961ac4f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg2ODQ0MzYsIm5iZiI6MTcyODY4NDEzNiwicGF0aCI6Ii8xODE5MDc0LzMzNjQzMDY4OC1lYTA0NTMwZi03MzlhLTQ4MzYtOTMyNi02MTc4MTk2MWFjNGYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTFUMjIwMjE2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzFjNDM5NTgyMTJkM2I1ZjlmOTAzZjU1MWMxM2JhNGZjOWJjZGIzZjZjMjhmYjk0NTgxN2ZjYmViOTg0ZmVhYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.8khIDsudaIH0OSjvVDvV0Ux_XGt5U2jQgjj1WBCmFKk)

EVERYONE, IT GOT FIXED!!
